src/app.js

```diff
@@ -44,7 +44,7 @@ app.use((req, res, next) => {
  */
 app.use((req, res, next) => {
   const user =
-    req.headers["x-forwarded-for"] || req.connection.remoteAddress || req.ip;
+    req.headers["x-forwarded-for"] || req.socket.remoteAddress || req.ip;
```

Comment:
Normally I just mention the depreciation here. an example of something included in the PR could be like 

> Here we replace `req.connection.remoteAddress` with `req.socket.remoteAddress` as  `req.connection.remoteAddress` is flagged as Deprecated in the [http package documentation](https://nodejs.org/api/http.html#http_request_connection) 
> request.connection is an alias of request.socket, you can directly use request.socket as a drop in replacement

with that added info above this change is perfect to add as is! This change is very useful and thanks again!